### ISSUE TITLE
Zoek API's template A11Y check

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#105]
**Deze problemen kunnen op meerdere pagina's aanwezig zijn, waaronder:**
[Zoek API's](https://rijkshuisstijl-community-templates-3losjges9-nl-design-system.vercel.app/developer-overheid/api-overzicht)

1. **Probleembeschrijving:**
De pagina heeft geen goede herindeling bij hoge zoomniveaus (controleer op 300% en hoger). De inhoud blijft horizontaal in plaats van dat een deel van de inhoud naar beneden wordt geduwd.

**Referentiecode:**
`<body class="rhc-theme">`

**Gerelateerde WCAG:**
WCAG 1.4.10 Reflow
Level AA

**Referentieafbeelding:**
![Zoomed in page where the elements are smushed together](https://github.com/user-attachments/assets/a0298748-85cc-46fa-a69d-3e5ddfa9ffd5)
![Zoomed in page where the card elements are smushed together with the filters](https://github.com/user-attachments/assets/23f4aedf-787e-4db3-bae1-4cbcdf56dc00)

_**Referentienummer:** TBD_

---

2. **Probleembeschrijving:**
Firefox Zoom Text Only Mode - Sommige afbeeldingen verliezen hun centrering binnen hun componenten wanneer er wordt ingezoomd.

**Referentiecode:**
`<span class="utrecht-icon" role="presentation" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-arrow-down "><path d="M12 5l0 14"></path><path d="M18 13l-6 6"></path><path d="M6 13l6 6"></path></svg></span>`

**Gerelateerde WCAG:**
WCAG 1.4.10 Reflow
Level AA

**Referentieafbeelding:**

![A button with the text Meer opties where the arrow graphic next to it is not centered](https://github.com/user-attachments/assets/8a3aa277-574e-4626-a78e-4fcc4e439ad8)
![A button with the text Meer opties where the plus graphic next to it is not centered](https://github.com/user-attachments/assets/13556472-4872-4693-a96b-81871c7507e4)
![A breadcrumb navigation where the greater than graphic next between the items is not centered](https://github.com/user-attachments/assets/fa281248-a062-4f8c-99aa-aeb0102946b9)

_**Referentienummer:** TBD_

---

3. **Probleembeschrijving:**
De Kaarten missen elementen met toegankelijke namen. De Koptekst, Organisatie en de pilvormige API-type labels zijn momenteel niet focusbaar en hebben geen toegankelijke naam. Dit betekent dat de focusvolgorde van de "+ API toevoegen" knop helemaal naar beneden springt naar de Footer, wat onverwacht is.

**Referentiecode:**
```
<div class="rhc-card__content"><div class="rhc-card__icon"></div><div class="rhc-card__heading">CBS Cijfers Open Data</div><div class="rhc-card__description">Centraal Bureau voor de Statistiek.</div><div class="rhc-card__metadata">Alle Nederlandstailige Statistieken van CBS ontsloten via oData versie 4</div><div class="utrecht-badge-list"><span class="utrecht-data-badge rhc-templates-databadge">OData</span></div></div>
```

**Gerelateerde WCAG:**
WCAG 2.4.3 Focus Order + 4.1.2 Name, Role, Value, 2.1.1 Keyboard
Level A

**Referentieafbeelding:**

![A widget shows the accessible names of the elements on the page. The Card elements have no accessible names.](https://github.com/user-attachments/assets/e736a52e-b3e3-4b35-a78c-1e0e491866b8)

_**Referentienummer:** TBD_
